src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2443,6 +2443,12 @@ internal BoundExpression GenerateConversionForAssignment(TypeSymbol targetType,
                     GenerateImplicitConversionError(diagnostics, expression.Syntax, conversion, expression, targetType);
                 }
 
+                if (expression.Kind == BoundKind.TupleLiteral)
```

Comment:
And not enter for invalid.

---
In reply to: [118418959](https://github.com/dotnet/roslyn/pull/19679#discussion_r118418959) [](ancestors = 118418959)